src/Firely.Fhir.Packages/Tar/Tar.cs

```diff
@@ -0,0 +1,235 @@
+﻿
+#if !NETSTANDARD1_6
+using ICSharpCode.SharpZipLib.GZip;
+using ICSharpCode.SharpZipLib.Tar;
+#endif
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Firely.Fhir.Packages
+{
+    public static class Tar
+    {
+        public static string PackToDisk(string path, IEnumerable<FileEntry> entries)
+        {
+#if !NETSTANDARD1_6
+            var packagefile = Path.ChangeExtension(path, ".tgz");
+
+            using var file = File.Create(packagefile);
+            using var gzip = new GZipOutputStream(file);
+            using TarOutputStream tar = new TarOutputStream(gzip);
+
+            Write(tar, entries);
+
+            return packagefile;
+#else
+            return "";
```

Comment:
To be uniform with the other methods, I would throw here also a `NotImplementedException`